content/en/docs/armory-admin/prometheus-monitoring.md

```diff
@@ -54,23 +59,241 @@ Access the Prometheus web interface by using the `kubectl port-forward` command.
+      - spin-gate
+      - spin-gate-api
+      - spin-gate-custom
+      - spin-deck
+      - spin-deck-custom
+      - spin-redis
+      - spin-terraformer
+      - spin-dinghy
+    matchLabels:
+      app: spin
+    namespaceSelector:
+      any: true
+
+```
+
+The example excludes Gate, the API service since Gate restricts access to the endpoints unless authenticated (excluding health).  
+
+The following example is for a service monitor for Gate on a different path and using TLS.
+
+Once these are applied, you can port forward prometheus and validate that prometheus
+has discovered and scraped targets as appropriate.
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: spinnaker-internal-metrics
+  namespace: spinnaker
+  labels:
+    app: spin
+    # this label is here to match the prometheus operator serviceMonitorSelector attribute
+    # prometheus.prometheusSpec.serviceMonitorSelector
+    # https://github.com/helm/charts/tree/master/stable/prometheus-operator
+    release: prometheus-operator
+spec:
+  selector:
+    matchLabels:
+      cluster: spin-gate
+      app.kubernetes.io/name: gate
+    namespaceSelector:
+      any: true
+  endpoints:
+  # "port" is string only. "targetPort" is integer or string.
+  - interval: 10s
+    path: "/api/v1/aop-prometheus"
+    scheme: "https"
+    tlsConfig:
+      insecureSkipVerify: true
+```
+## Check for Spinnaker targets in Prometheus
+
+After applying these changes, you should be able to see  Spinnaker targets in Prometheus. It may take 3 to 5 minutes for this to show up depending on where Prometheus is in its config polling interval.
+
+![Prometheus Targets](/images/prometheus-2.png)
+
+## Access Grafana
+
+Configure port forwarding for Grafana:
+
+```bash
+$ kubectl --namespace monitoring port-forward svc/grafana 3000
+```
+
+Access the Grafana web interface via http://localhost:3000 and use the default grafana user:password of `admin:admin`.
+
+## Add Armory dashboards to Grafana
+
+Armory provides some sample dashboards (in JSON format) that you can import into Grafana as a starting point for metrics to graph for monitoring. Armory has additional dashboards that are availabe to Armory customers. You can skip this section if you are a Grafana expert.
 
-To enable monitoring of Spinnaker by Prometheus, enable the `metric-stores` configuration.
+To import the sample dashboards, perform the following steps:
+
+1. Git clone this repo to your local workstation: (https://gitlab.com/uneeq-oss/spinnaker-mixin)
+2. Access the Grafana web interface (as shown above)
+3. Navigate to Dashboards then Manage
+4. Click on the _Import_ button
+5. Upload the one or more of the sample dashboard files from the repo you cloned
+
+After importing the dashboards, you can explore graphs for each service by clicking on **Dashboards > Manage > Spinnaker Kubernetes Details**.
+
+![Grafana Dashboard](/images/grafana-2.png)
+
+## Configure monitoring using the monitoring-daemon
```

Comment:
@christosarvanitis can we remove everything from here until the end of the file since monitoring-daemon is no longer supported by Armory and we recommend the Observability plugin?

Reply:
@brian-armory sure, if you prefer to completely remove any mention to the monitoring-daemon then we can remove